main.go

```diff
@@ -39,6 +39,7 @@ const (
 		`                        connect:%s         |                  |` + "\n" +
 		`                                      starttransfer:%s        |` + "\n" +
 		`                                                                 total:%s` + "\n"
+	HTTPSTAT_SHOW_BODY = "HTTPSTAT_SHOW_BODY"
```

Comment:
I don't think we need to a constant here, the constant isn't any shorter or more descriptive than it's value.